hotel_elasticsearch/app.py

```diff
@@ -88,13 +133,30 @@ def run():
     if not service.running():
         service.start()
 
-    webserver = Thread(target=app.run, )
-    webserver.daemon = True
-    webserver.start()
-
-    backup = Thread(target=backup_thread, )
+    # Install any required templates only on the first run
+    if cluster_node.tags.get('initial_master', 'False') == 'True':
+        try:
+            for file in os.listdir('/etc/hotel-elasticsearch/templates'):
+                if file.endswith('.json'):
+                    with open('/etc/hotel-elasticsearch/templates/' + file) as f:
+                        response = requests.put(
+                            'http://localhost:9200/_template/' + file[:-5],
+                            json.dumps(file.read()),
+                            headers={'Content-Type': 'application/json'},
+                            params={'include_type_name': 'true'},
+                        )
+                        response.raise_for_status()
+
+        except FileNotFoundError():
+            pass
```

Comment:
log some warning?

Reply:
No I don't think so. Dependent on your use case you might not want to make template installation part of bootstrapping a cluster